jobs/nightly-rpm-builds.yml

```diff
@@ -49,17 +58,17 @@
         - project: gluster_build-rpms
           block: false
           predefined-parameters:
-            GERRIT_BRANCH=release-6
+            GERRIT_BRANCH=release-8
 
-            CENTOS_VERSION=7
+            CENTOS_VERSION=8
 
             CENTOS_ARCH=x86_64
     - trigger-builds:
         - project: gluster_build-rpms
           block: false
           predefined-parameters:
-            GERRIT_BRANCH=release-6
+            GERRIT_BRANCH=release-7
```

Comment:
Any specific reason to not build release-7 branch on other centos versions?

Reply:
I was not sure if we have enough build machines to do that. Now Included other centos versions for building release-7